usercp.php

```diff
@@ -3539,16 +3539,16 @@
 	$usage = $db->fetch_array($query);
 	$totalusage = $usage['ausage'];
 	$totalattachments = $usage['acount'];
-	$friendlyusage = get_friendly_size($totalusage);
+	$friendlyusage = get_friendly_size((int)$totalusage);
 	if($mybb->usergroup['attachquota'])
 	{
-		$percent = round(($totalusage/($mybb->usergroup['attachquota']*1024))*100)."%";
+		$percent = "(".round(($totalusage/($mybb->usergroup['attachquota']*1024))*100)."%)";
```

Comment:
I'd rather not add anything hardcoded. Please wrap this value in brackets in the language files.

Reply:
It would be appreciated if you could add both percent and brackets to $lang->attachments_usage_quota. It's not used anywhere else, so there's no need to worry about messing up something else.

Reply:
@Shade- 
$lang->attachments_usage_quota is being used at 'thead' of attachment manager.
Added new language variable and removed duplicate brackets from quota.

Done.

Reply:
> $lang->attachments_usage_quota is being used at 'thead' of attachment manager.

This is what you are editing in this file. It's not used anywhere else, there's no need to create a separate language variable.

Reply:
No,
I am editing the value of "Space Used" under "Your Attachment Statistics". There is a (##%) beside used size which doesn't show %, but unnecessary Quota limit as (Unlimited) while there is no quota restriction.

Reply:
@Shade- can you please check what I said and suggest?